seb_openedx/models.py

```diff
@@ -0,0 +1,20 @@
+""" Models """
+from __future__ import absolute_import, unicode_literals
+from django.db import models
+from opaque_keys.edx.django.models import CourseKeyField
+
+
+class ForbiddenCourseAccess(models.Model):
+    """ user_id <-> course_id relation to mark forbidden users """
+    username = models.CharField(max_length=150)  # from django.contri.auth.user
```

Comment:
In theory a user should not change its username, so this is not so problematic, but I still think the user_id is safer

Reply:
user_id doesn't exist in the User model, it's that something by edx?

Reply:
at the user model is called user.id, but it is the same